test/router.ts

```diff
@@ -0,0 +1,317 @@
+var assert = require("assert");
+var router = require("../lib/router");
+
+describe("router.Router", () => {
+  describe("constructor", () => {
+    it("parses well formatted configs", () => {
+      process.env.SCHOOL = "Hogwarts";
+      const conf = `
+routes:
```

Comment:
Sounds good! We should make sure it has a version in it, which I think I've overheard you discussing.